web/templates/web/pages/top_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends "web/pages/base.html" %}
+
+{% load static %}
+
+{% block title %}
+  Топ лист
+{% endblock title %}
+{% block styles %}
+  <link rel="stylesheet" href="{% static 'web/css/top_list.css' %}?">
+{% endblock styles %}
+{% block body %}
+  {% include "web/includes/tab.html" %}
+  <div class="container">
+    <div class="is-flex is-flex-direction-row is-justify-content-end">
+      <div class="is-flex buttons has-addons ">
+        <button id="top_list_button" class="button is-selected-switch-button">
+          ТОП-3
+        </button>
+        <button id="artist_list_button" class="button">Список</button>
+      </div>
+    </div>
+    <div id="top_list_content">
+      <div class="columns is-centered is-vcentered">
+        {% for artist in top_artists|slice:":3" %}
+          <div class="column is-display-flex is-flex-direction-column is-align-items-center">
+            <figure class="image is-300x300">
+              <img class="is-rounded"
+                   src="{{ artist.artists__avatar }}400x400"
+                   alt="Artist image" />
```

Comment:
Лучше на русском писать `alt`ы, потому что весь сайт на русском языке, а `alt` отображается в виде текста вместо картинки. Аналогично на 56 строчке
```suggestion
                   alt="Фото артиста" />
```